utils/kubernetes/container.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2023 Red Hat, Inc. and/or its affiliates
```

Comment:
maybe reneme this file to security_context.go?

Reply:
ok

Reply:
renamed